kbs/src/http/resource.rs

```diff
@@ -113,12 +114,25 @@ pub(crate) async fn get_resource(
         info!("Resource access request passes policy check.");
     }
 
-    let resource_byte = repository
-        .read()
-        .await
-        .read_secret_resource(resource_description)
-        .await
-        .map_err(|e| Error::ReadSecretFailed(format!("{e:?}")))?;
+    let resource_byte = if resource_description.repository_name == "plugin" {
+        repository_plugin
+            .read()
+            .await
+            .get_resource(
+                resource_description.resource_type.as_str(),
+                resource_description.resource_tag.as_str(),
+                request.query_string(),
+            )
+            .await
+            .map_err(|e| Error::ReadSecretFailed(e.to_string()))?
+    } else {
+        repository
+            .read()
+            .await
+            .read_secret_resource(resource_description)
+            .await
+            .map_err(|e| Error::ReadSecretFailed(e.to_string()))?
+    };
```

Comment:
We also need to update the KBS PROTOCOL document under [docs](https://github.com/confidential-containers/trustee/blob/main/kbs/docs/kbs_attestation_protocol.md) to show that `plugin` is reserved.